analysis/monitor.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+
+from matplotlib.pyplot import figure, show
+import datetime
+import dateutil.parser
+import json
+import numpy as npy
+import os
+import pylab as plt
+import sys
+import urllib
+
+if (len(sys.argv) < 2):
+    print "Usage: " + sys.argv[0] + " captor_id"
+    print
+    print " `-> Prints the hours when too many measures were took for the captor whose id was given in argument."
+    print "     A graph is built with one curve by day displaying the number of measures by hour"
+    print
+    exit()
+
+captor = sys.argv[1]
+
+secret_json = os.path.dirname(os.path.abspath(__file__)) + "/../PRIVATE.json"
+with open(secret_json) as configuration_json:
+    configuration = json.load(configuration_json)
+
+url = configuration["data_source"] + "/measurements/places?ids=" + captor + "&types=wifi"
+measures = json.loads(urllib.urlopen(url).read())
+measures.sort(key=lambda arr: arr["date"])
+
+X = range(0, 24)
+res = X[:]
+last = datetime.datetime.fromtimestamp(0)
+before = last
+for i in range(0, len(X)):
+    res[i] = 0
+
+for i in range(0, len(measures)):
```

Comment:
In python, to iterate through arrays you can use:
```
for index, element in enumerate(array):
    print 'This is element %s at position %i' % (str(element), index)
```
Thus, you can use `element` instead of `array[i]`.